plugins/pipelines/create.js

```diff
@@ -31,15 +31,15 @@ module.exports = (server, options) => ({
                 }, next),
                 (permissions, next) => {
                     if (!permissions.admin) {
-                        return reply(boom.unauthorized(`User ${username} `
+                        return next(boom.unauthorized(`User ${username} `
                             + 'is not an admin of this repo'));
                     }
 
                     return Pipeline.get(pipelineId, next);
                 },
                 (data, next) => {
```

Comment:
We're not leveraging the data for anything other than whether it's truthy or falsy. Could you update the variable to better describe what you'll use it for? 

In this specific case, you're using it to check whether a pipeline exists for a given SCM url. Renaming it to something along the lines of `doesPipelineExist` would make it much more clearer as to what significance it has for the function.